### DISCUSSION TITLE
How do I define a module as double track?

### DISCUSSION THREAD

**Discussion by AlexEhn** [tellurianinteractive/Tellurian.Trains.ModulesRegistryApp#119] [Q&A] [answered]
This is a MMM Double track module. Should I first create a MMM4802R Double track gable?

Or is it the Number of through tracks that is supposed to capture that? 
> Number of Through Tracks is how many parallel tracks there are: 1 = single track, 2=double track etc. You also specify this for end modules like shadow stations, even if the tracks are not through.

However, if I have through tracks that is non-standard cc-measurement, that cannot be captured.
Does the parameter Through track assumes that all through tracks are standard-compliant? 
How do I capture a 2-1 module?

![image](https://user-images.githubusercontent.com/57874863/131731754-a17fd282-7244-4447-a4f6-42238bfe00f6.png)

**Comment by fjallemark** [comment]
Use the **Number of through tracks**:
- One through track means a single track module.
- Two through tracks means a double track module.

Modules with more that two through tracks exists, but rather seldom and for special layouts.

This was missing in the module help page, but will be added in the next release.

**Reply by AlexEhn** [reply]
Ok, good. 

And the implication of that is that tracks within a module that goes between sections are irrelevant to capture from a registry perspective.

Only standard compliant gables are of interest. If it's not standard it should not be registered. Correct?

**Comment by fjallemark** [comment]
Yes. Only *logical* modules are of interest. If a *logical* module consists of one or several physical modules is of little interest, and consequently the module joints within a *logical* module is irrelevant.

**Comment by AlexEhn** [comment]
The description "logical module" (logisk modul) does to mean anything to me. What is is? 
I think I know what you mean, as in module vs. module sections?

**Comment by fjallemark** [comment, marked as answer]
Well, i used *module* for your *module section* and *logical module* for your *module*. But **module** and its **module sections** is probably the right terminology.

**Comment by AlexEhn** [comment]
Perfect. Thanks!
